go-runtime/compile/schema.go

```diff
@@ -508,7 +508,7 @@ func visitComments(doc *ast.CommentGroup) []string {
 	return comments
 }
 
-func visitStruct(pctx *parseContext, node ast.Node, tnode types.Type) (*schema.Ref, error) {
+func visitStruct(pctx *parseContext, pos token.Pos, tnode types.Type) (*schema.Ref, error) {
 	named, ok := tnode.(*types.Named)
 	if !ok {
 		return nil, fmt.Errorf("expected named type but got %s", tnode)
```

Comment:
The raw fmt.Errorf() calls in this function should now use errorf() if they can?